limitlion/running_counter.py

```diff
@@ -255,6 +254,26 @@ def group_counts(self, recent_buckets=None):
 
         return values
 
+    def group_buckets_counts(self, recent_buckets=None):
+        """
+        Get count for each counter and bucket in group. Missing buckets are
+        filled with 0. Most recent buckets are first.
```

Comment:
Nit: Seems like "Missing buckets are filled with 0. Most recent buckets are first." are both implementation details of `buckets_counts`. I would change this docstring to just "Get counts for each counter and bucket in a group.".

Reply:
Yea that's fair, it was last minute change, and user can always check the implementation, so will remove.